Packet++/src/PacketUtils.cpp

```diff
@@ -1,4 +1,8 @@
+#ifndef _WIN32
+#include <arpa/inet.h>

```

Comment:
perhaps use the macros from EndianPortable.h instead?
Its already included a few lines below.

Reply:
true